dndwiki/races/race-dryad.xml

```diff
@@ -1,96 +1,118 @@
+			<grant type="Racial Trait" id="ID_WIKI_RACIAL_TRAIT_DRYAD_TREE_STRIDE" />
+		</rules>
+	</element>
+
+	<element name="Blend In" type="Racial Trait" source="D&amp;D Wiki" id="ID_WIKI_RACIAL_TRAIT_DRYAD_BLEND_IN">
+		<description>
+			<p>You have advantage on Stealth checks made to hide amongst foliage or near a huge group of trees.</p>
+		</description>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Dryad_(5e_Race)&amp;oldid=954506#Dryad_Traits</set>
+		</setters>
+		<sheet>
+			<description>You have advantage on Stealth checks made to hide amongst foliage or near a huge group of trees.</description>
+		</sheet>
+	</element>
+
+	<element name="Nature Whisperer" type="Racial Trait" source="D&amp;D Wiki" id="ID_WIKI_RACIAL_TRAIT_DRYAD_NATURE_WHISPERER">
+		<description>
+			<p>After living in the forest for your whole life you can communicate with beasts and plants as if they share a language.</p>
+		</description>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Dryad_(5e_Race)&amp;oldid=954506#Dryad_Traits</set>
+		</setters>
+		<sheet>
+			<description>You can communicate with beasts and plants as if they share a language.</description>
+		</sheet>
+	</element>
+
+	<element name="Photosynthesis" type="Racial Trait" source="D&amp;D Wiki" id="ID_WIKI_RACIAL_TRAIT_DRYAD_PHOTOSYNTHESIS">
+		<description>
+			<p>Being a plant based life form, you create your food through Photosynthesis, being in sunlight will regenerate 1d4 health per hour, likewise, if you aren't in sunlight for two days you will start to decay.</p>
+		</description>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Dryad_(5e_Race)&amp;oldid=954506#Dryad_Traits</set>
+		</setters>
+		<sheet>
+			<description>Being in sunlight will regenerate 1d4 hit points per hour, likewise, if you aren't in sunlight for two days you will start to decay.</description>
+		</sheet>
+	</element>
+
+	<element name="Tree Stride" type="Racial Trait" source="D&amp;D Wiki" id="ID_WIKI_RACIAL_TRAIT_DRYAD_TREE_STRIDE">
+		<description>
+			<p>Over time you have learned the skill of using trees as a type of transportation. Once a turn, You can use 10 feet of your movement to step magically into one living tree within your reach and emerge from a second living tree within 60 feet of the first tree, appearing in an unoccupied space within 5 feet of the second tree. Both trees must be Large or bigger.</p>
+		</description>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Dryad_(5e_Race)&amp;oldid=954506#Dryad_Traits</set>
+		</setters>
+		<sheet usage="1/Turn">
+			<description>You can use 10 feet of your movement to step magically into one living tree within your reach and emerge from a second living tree within 60 feet of the first tree, appearing in an unoccupied space within 5 feet of the second tree.</description>
```

Comment:
"Both trees must be Large or bigger." will be a nice add on for clarification for players.